<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musPlayer</class>
 <widget class="QWidget" name="musPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1588</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>850</y>
     <width>951</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="progressSlider">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLast">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBegin">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
